src/Illuminate/Database/Query/Builder.php

```diff
@@ -2637,6 +2649,18 @@ public function limit($value)
         return $this;
     }
 
+    /**
+     * Get the current "limit" value from the query or null if not set.
+     *
+     * @return mixed
+     */
+    public function getLimit()
```

Comment:
Why not 

    /**
     * Get the current "limit" value from the query or null if not set.
     */
    public function getLimit() ?int

analog for all NEW function declarations. Dockblocks are not needed if you write the function definition with return type and typed params.

Reply:
Just matching the same style as is in there right now

Reply:
This is not in compliance with what laravel stated that starting with laravel 10, that they will implement typed declarations...
I see many new functions merged with old style codding...